Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/list/VaccinationListViewModel.kt

```diff
@@ -125,6 +144,12 @@ class VaccinationListViewModel @AssistedInject constructor(
         events.postValue(Event.NavigateToVaccinationQrCodeScanScreen)
     }
 
+    fun deleteVaccination(vaccinationCertificateId: String) {
+        launch(scope = appScope) {
+            vaccinationRepository.deleteVaccinationCertificate(vaccinationCertificateId)
```

Comment:
delete can throw an exception ,right?  if so please catch it and report the error?

Reply:
Yes, fixed.